_summaries/wang2018dynamic_1.md

```diff
@@ -1,19 +1,95 @@
+
+
+// TODO - put the formal definition of EdgeConv here
+
+
+## Okay... but why is it called "Dynamic Graph"
+
+- Once EdgeConv has ran on the point cloud, you now have a bunch of new values for the point cloud that have been mutated.
+
+- Empirically, the authors found that it is beneficial to recompute the K nearest neighbours after each edge conv.
+
+- This leads to points in the cloud becoming closer due to their feature space similarity not despite their input proximity!
+
+- This recomputation of the K nearest neighbour is a reconstruction of the graph and thus influenced the authors to name their method "Dynamic Graph CNN"
+
+
+## How is this different from PointNet & PointNet++
+
+- PointNet is a simplification of EdgeConv where you ignore the neighbourhood of the K nearest datapoints and their features
+
+- PointNet++ does not change the proximity of points in the cloud based on feature simularity (there graph is fixed)
+  - Further, PointNet++ looks at the neighbourhoods edges then takes the max feature from them
+  
+## Results of DGCNN 
+
+- Most comparisons are done on classification of point clouds via the architecture seen in the image below.
+
+![Specific Classification Archeticture](wang2018dynamic_1d.png)
+
+- With static graphs (similar to PointNet++, input proximity is saved throughout the feature space) DGCNN already outperforms PointNet++
+
+- With a new distance function that looks at global and local structure, DGCNN outperforms all models on ModelNet40 (dramatically with more points!)
+
+- DGCNN is faster than PointNet++ (slower than PointNet)
+
+![Classification Results](wang2018dynamic_1e.png)
+
+- DGCNN can also segment objects, although it is not state of the art in its results, it is able to find similarities across various objects
+
+![Qualitative Segmentation Results](wang2018dynamic_1f.png)
+
+- DGCNN finds similar regions on unseen objects (right) from trained on point clouds (left).
+
+
+## What are the limitations?
+
+![Dependency on Dense Data](wang2018dynamic_1a.png)
+
+- DGCNN requires consistently dense data to perform well, sparse data will have a large drop off in accuracy
+
+- Because of the requirement on dense data, it can be very memory and computationally expensive
+
+- Again because of the requirement of dense data, DGCNN does not reduce the number of data points per layer like other point cloud algorithms (consistently dense input)
+
+![K Hyperparameter can hurt performance](wang2018dynamic_1g.png)
+
+- The hyperparameter K (how large are the neighbourhoods) is fickle, and can lead to impaired performance if too high
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* EdgeConv takes 1 vertex, then compares it through some metric to a bunch of other vertices, which then go through a nonlinear transform then is aggregated with the other neighbouring vertices.
```

Comment:
Maybe this point can be a bit more detailed, since other point cloud methods could also compute "some metric to a bunch of other vertices".

Reply:
"EdgeConv computes all edges from one vertex to a neighborhood of other vertices, including an edge to the same vertex, these edge values are then aggregated updating the original value of the vertex."

I reworded it.  I am trying to leave out a specific metric because they say you could really use anything.  Dot Product, Euclidean distances, etc. that's how you can end up recreating pointnet/pointnet++ with their implementation.  But it's hard to really sum that up in one nice bullet point.  I think the updated wording gets closer though